tests/cypress/integration/components/a-z-nav.js

```diff
@@ -0,0 +1,46 @@
+describe('setting screen res', function () {
+
+  beforeEach(function () {
+    cy.viewport('macbook-15');
+
+  });
+
+  it('Navigating to A-Z Navigation', function () {
+
+    cy
+      .visit(Cypress.config().baseUrl + '/nav-a-z.html')
+
+  });
+
+
+  // checking for class name - nhsuk-c-a-z-nav__list
```

Comment:
Do we need comments here? As it is described in plain english in the `it('Checking for class name - nhsuk-c-a-z-nav__list'`

Reply:
removed the comments